master/custom/builders.py

```diff
@@ -0,0 +1,141 @@
+from master.custom.factories import (
+    UnixBuild,
+    UnixRefleakBuild,
+    UnixInstalledBuild,
+    AIXBuild,
+    AIXBuildWithoutComputedGotos,
+    NonDebugUnixBuild,
+    PGOUnixBuild,
+    ClangUbsanLinuxBuild,
+    ClangUnixInstalledBuild,
+    SharedUnixBuild,
+    LTONonDebugUnixBuild,
+    LTOPGONonDebugBuild,
+    WindowsBuild,
+    SlowWindowsBuild,
+    Windows27VS9Build,
+    Windows6427VS9Build,
+    Windows64Build,
+    Windows64RefleakBuild,
+    Windows64ReleaseBuild,
+    WindowsArm32Build,
+    WindowsArm32ReleaseBuild,
+)
+
+STABLE = "stable"
+UNSTABLE = "unstable"
+
+
+# classes using longer timeout for koobs's very slow buildbots
+class SlowNonDebugUnixBuild(NonDebugUnixBuild):
+    test_timeout = 30 * 60
+
+
+class SlowSharedUnixBuild(SharedUnixBuild):
+    test_timeout = 30 * 60
+
+
+# We use one scheduler per branch (as the AnyBranchScheduler is useless)
+# and one builder per worker and branch
+
+# the 'builders' list defines the Builders. Each one is configured with a
+# dictionary, using the following keys:
+#  name (required): the name used to describe this builder
+#  workername (required): which worker to use, must appear in c['bots']
+#  builddir (required): which subdirectory to run the builder in
+#  factory (required): a BuildFactory to define how the build is run
+#  periodicBuildTime (optional): if set, force a build every N seconds
```

Comment:
These two comments can probably just go away.  Neither one makes a whole lot of sense here anymore.